packages/devtools-launchpad/src/actions/config.js

```diff
@@ -1,7 +1,5 @@
-/* This Source Code Form is subject to the terms of the Mozilla Public
- * License, v. 2.0. If a copy of the MPL was not distributed with this
- * file, You can obtain one at http://mozilla.org/MPL/2.0/. */
-/* global window */
+const { setConfig: _setConfig } = require("devtools-config");
```

Comment:
I think its because we have an action defined in the file already called `setConfig`